qgepplugin/processing_provider/provider.py

```diff
@@ -25,6 +25,9 @@
 from PyQt5.QtGui import QIcon
 from qgis.core import QgsProcessingProvider
 
+from ..qgepqwat2ili.qgepqwat2ili.processing_algs.extractlabels_interlis import (
```

Comment:
shouldn't we use absolute imports with qgepplugin.qgepqwat2ili ?

Reply:
Here I think it's fine, because qgepqwat2ili isn't installed in python path (it's also imported relatively in other places). IMO safer so we avoid any conflict if users also have pip installed qgepqwat2ili separately from the plugin.

Of course this is all a bit dirty, since there's still no clean way to manage pip dependencies for python plugins...